code/api/src/migrations/8-shipments.js

```diff
@@ -17,10 +17,10 @@ module.exports = {
         },
         allowNull: false
       },
-      crateId: {
+      subscriptionId: {
```

Comment:
This will allow us to have access to not only the subscription but also the crate - nice change

Reply:
Agreed. This is a great edit to keep it more dynamic.